Pedigree editor fixes: LFS-605, LFS-1172, LFS-1173

marta- opened the pull request:
Includes:
- [x] LFS-605: Pedigree editor: Export and Import dialogs are sometimes misplaced
- [x] LFS-1172: Unable to save Disorders and Phenotypic features in Pedigree editor
- [x] LFS-1173: When attempting to leave the pedigree without saving the changes the screen gets darker but nothing else appears to happen

marta- committed:
LFS-1172: Unable to save Disorders and Phenotypic features in Pedigree editor

sashaandjic reviewed (approved):
tested
![image](https://user-images.githubusercontent.com/23243244/119735960-18a19780-be4b-11eb-835b-d8d08afbbb55.png)

marta- committed:
LFS-1173: When attempting to leave the pedigree without saving the changes the screen gets darker but nothing else appears to happen
Quick fix to bring the "Are you sure...?" dialog within view and improve the display of its buttons

marta- committed:
LFS-605: Export and Import dialogs are sometimes misplaced

sashaandjic reviewed (approved):
tested 
![image](https://user-images.githubusercontent.com/23243244/119740664-4b9b5980-be52-11eb-9cd8-6628da926685.png)

sashaandjic commented:
- [ ] LFS-1172
- [ ] LFS-1173
- [ ] LFS-605
![image](https://user-images.githubusercontent.com/23243244/119742172-2cea9200-be55-11eb-8e79-9844ed2b9f48.png)
![image](https://user-images.githubusercontent.com/23243244/119742186-370c9080-be55-11eb-986f-9ba50a9edf73.png)
